### PR TITLE
Aggregate appointments

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_Appointments.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+IF OBJECT_ID(N'[REPORTING].[dbo].AggregateAppointments', N'U') IS NOT NULL 		
+	truncate table [REPORTING].[dbo].AggregateAppointments
+GO
+
+Insert into [REPORTING].[dbo].AggregateAppointments
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_Appointments.sql:5]: @Marymary-dev should we also add AgeGroup or that can come later ?

**Comment by Marymary-dev** [comment]
Alright, this has also been added

File: Scripts/REPORTING/load_aggregate_Appointments.sql (hunk 1 above)
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_Appointments.sql:5]: @nobert-mumo  have added the DATIM Age Group slicer, Gender is already there

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_Appointments.sql:5]: LGTM!

**Review by nobert-mumo — APPROVED** [review]

**Pull request merged**